DirectEmployers/MyJobs
# Allow archiving of OutreachEmailAddresses [PD-3735]

## What's this PR do?
Adds ArchivedModel superclass to OutreachEmailAddress
## Screenshots (if appropriate)
N/A
## Where should the reviewer start?
The various api views would be a good place.
## How should this be manually tested?
Have an outreach address that has records associated with it. Delete it. Verify that you still see your records. Create another identical address.
## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
